AJS.Web/Resources/Areas/Identity/Pages/Account/ResetPasswordModel.bg.resx

```diff
@@ -0,0 +1,124 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<root>
+  <!-- 
+    Microsoft ResX Schema 
+    
```

Comment:
Must be Localized "Reset your password" in ResetPassword View!
`<h4>Reset your password.</h4>`